src/data-access/impl.ts

```diff
@@ -171,13 +185,36 @@ export class ArchiverDataAccess implements DataAccess {
             `Request returned an empty array, probably due to an invalid range for the url ${jsonurl}`
           );
         }
-
         return res.data[0];
       });
 
+    finalData = this.parseData(res.data);
+
+    //Encontrar valor do tempo, conseguir seu index e retirar o valor para fazer o diff
+
+    if(diff == true){
+      const selectedDate = new Date(sessionStorage.getItem('selectedDate'));
+
+      let closestDate = 100000000000000000000000000000000000000000;
```

Comment:
cannot you use scientific notation here? or better yet, a language constant that represent a maximum type value?